Makefile

```diff
@@ -406,7 +406,7 @@ docker-host-dir-setup:
 	$(DOCKER_COMPOSE) build --no-cache koku-minio
 	mkdir -p -m 0755 $(CONTAINER_DIRS) 2>&1 > /dev/null
 	chown -R $(USER_ID) $(CONTAINER_DIRS)
-	chmod 0755 $(CONTAINER_DIRS)
+	chmod -R 0755 $(CONTAINER_DIRS)
```

Comment:
We will need to create an empty subdir within `pg_data`
```makefile
CONTAINER_DIRS = $(TOPDIR)/pg_data/data $(TOPDIR)/.trino/{parquet_data,trino}
```

Reply:
Also, the  data mount in the `docker-compose.yml` file needs to change.

```patch
diff --git docker-compose.yml docker-compose.yml
index 430062cf..1546ce7a 100644
--- docker-compose.yml
+++ docker-compose.yml
@@ -452,7 +452,7 @@ services:
     ports:
       - 15432:5432
     volumes:
-      - ./pg_data:/var/lib/${DATABASE_DATA_DIR-postgresql}
+      - ./pg_data/data:/var/lib/${DATABASE_DATA_DIR-postgresql}/data
       - ./pg_init:/docker-entrypoint-initdb.d
     command:
       # This command give more precise control over the parameter settings

```

Reply:
This note from the [readme](https://hub.docker.com/_/postgres) explains what's going on:

> If the data volume you're using is a filesystem mountpoint (like with GCE persistent disks), or remote folder that cannot be chowned to the postgres user (like some NFS mounts), or contains folders/files (e.g. lost+found), Postgres initdb requires a subdirectory to be created within the mountpoint to contain the data.